pybsn/__init__.py

```diff
@@ -467,6 +468,16 @@ def __repr__(self):
         return "BigDbClient(%s)" % self.url
 
 
+def _normalize(v):
+    """ Helper method to normalize query values """
+    if type(v) == bool:
+        # replace to use JSON type booleans, not Python
```

Comment:
nit: should maybe adjust the comment as well?

Reply:
Done